DictionaryTypes.txt

```diff
@@ -301,8 +301,11 @@ enum(coil) "Sensor coil type"
     fnirs_fd_phase             305  "fNIRS frequency domain phase"
     fnirs_td_gated_amplitude   306 "fNIRS time domain gated amplitude"
     fnirs_td_moments_amplitude 307 "fNIRS time domain moments amplitude"
-    eyetrack_pos    400  "Eye-tracking gaze position"
-    eyetrack_pupil  401  "Eye-tracking pupil size"
+    eyetrack_pos_deg    		400  "Eye-tracking gaze position (in DEG vis angle)"
+    eyetrack_pos_px    		401  "Eye-tracking gaze position (in PX on screen)"
+    eyetrack_pupil_area_au  	410  "Eye-tracking pupil size (area, in AU)"
+    eyetrack_pupil_diameter_au  	420  "Eye-tracking pupil size (diameter, in AU)"
+    eyetrack_pupil_diameter_mm  	421  "Eye-tracking pupil size (diameter, in MM)"
```

Comment:
Just leaving a note here, we will not use this coil type in the current PR, and we should change this to Meters, because meter is the SI unit of length, which we would convert pupil size to (if possible/feasbile).